src/backend/distributed/planner/multi_explain.c

```diff
@@ -1201,8 +1202,26 @@ CitusExplainOneQuery(Query *query, int cursorOptions, IntoClause *into,
 
 	INSTR_TIME_SET_CURRENT(planstart);
 
-	/* plan the query */
-	PlannedStmt *plan = pg_plan_query_compat(query, NULL, cursorOptions, params);
+	/*
+	 * we should not hide any objects while explaining some query to not break
```

Comment:
There are postgres tests which use explain commands querying some pg meta classes. Those tests fail with different plan results because we intercept pg meta class queries and modify the filter.